src/components/public/landing/hero-section.tsx

```
@@ -0,0 +1,187 @@
+    name: "Campus Bike",
+    description: "Get around campus quickly and easily",
+    price: 249.99,
+    discount: "5% OFF",
+    image: "/placeholder.svg?height=600&width=600&text=Campus+Bike",
+  },
+];
+
+const ColorCircle = React.memo(({ delay }: { delay: number }) => (
+  <motion.circle
+    r={Math.random() * 100 + 50}
+    cx={Math.random() * 100 + "%"}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)